sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -217,6 +217,8 @@ SHSetIniStringUTF7W(
     _In_opt_z_ LPCWSTR lpString,
     _In_z_ LPCWSTR lpFileName);
 
+HKEY WINAPI SHGetShellKey(DWORD flags, LPCWSTR sub_key, BOOL create);
```

Comment:
Note that you can now remove the declaration made in this file:
```
modules/rostests/apitests/shell32/ShellState.cpp	
  75 extern "C" HKEY WINAPI SHGetShellKey(DWORD flags, LPCWSTR sub_key, BOOL create);
```

Reply:
Fixed.